lib/element_mapping.js

```diff
@@ -166,7 +166,11 @@ module.exports = {
     noReplacement: true,
   },
   'core-list': {
-    noReplacement: true,
+    name: 'iron-list',
+    notice: 'iron-list no longer tracks selection of its elements. Also the ' +
+            'data binding of the template child has changed some to be more ' +
+            'like dom-repeat. See: ' +
```

Comment:
mention adding `as="model"` to fix bindings. Add TODO to add that automatically.